### DISCUSSION TITLE
Pull requests

### DISCUSSION THREAD

**Discussion by herman-playbk** [community/community#30495] [Pull Requests]
Would love to see my PR assignments and mentions here as well

**Comment by marcopollivier** [comment]
Yeah. Me too. 

Most of the time the notification center is noisy. I'd like to be able to see the PRs open to me and by me on the timeline easily

**Comment by andrewpmartinez** [comment]
As a maintainer of a project that has many repos, the lack of ability to see all my PRs and PRs waiting for review is highly disruptive. 

Maybe I'm missing where to find the list now, but I have to go to each repo and check them. The notification page is useless and cluttered; the home page feed is useless.

I don't know what Github is doing, but it makes me hate Github.
